### DISCUSSION TITLE
itzg / docker-minecraft-bedrock-server on Pterodactyl.io ---need nelp!!

### DISCUSSION THREAD

**Discussion by ShoNuffBadd** [itzg/docker-minecraft-bedrock-server#347] [Q&A]
is there any way or has someone setup this container/docker on Pterodactyl.io panel? I went through the hassle of setting up Pterodactyl on my ubuntu server and the "eggs" provided for use on Pterodactyl do not work. I was hoping someone might have used this docker image on there and have a working bedrock vanilla server. any help  would be appreciated.

**Comment by itzg** [comment]
They seem to have certain facilities and expectations about custom eggs that may not align well with my image's self sufficient startup mechanism. They allude to Minecraft setup here, so perhaps contact Pterodactyl for more help

https://pterodactyl.io/community/config/eggs/creating_a_custom_egg.html#list-of-default-variables
